aws/policy/compute.yaml

```diff
@@ -68,6 +68,7 @@ Statement:
           - t2.micro
           - t3.nano
           - t3.micro
+          - m1.large
```

Comment:
Is there a reason we can't use `t3a.nano` instead? It is listed as EBS-optimized by default:

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSOptimized.html

Reply:
Also, I see `t3.nano` is also listed there, in a second table under the same heading, although it does cost slightly more than the `t3a.nano` instances.

Reply:
It's because the instances listed in that table have EBS enabled by default and disabling it, won't have any effect.

> There is no need to enable EBS optimization and no effect if you disable EBS optimization

The instances we need to test are the ones listed in the **EBS Optimization Supported** table

Reply:
Thanks for the clarification. It looks like all the EBS Optimization Supported instances are previous generation instances. Do you know where the pricing is for those?

Reply:
Yeah, you can find them here: https://aws.amazon.com/ec2/previous-generation/

Reply:
Ah, I had looked there, but didn't see `m1.large` because I was looking at the wrong region.

It does appear to be the lowest cost, at $0.175/hour in us-east-1.